.github/workflows/ci_cd.yml

```
@@ -131,7 +131,8 @@ jobs:
         mapdl-version: [
           'v23.1-ubuntu',
           'v23.2.0',
-          'v24.1-ubuntu-student',
+          'v24.1-ubuntu',
+          'v24.2-ubuntu',
```

Maybe you can add newer versions here. 
```suggestion
          'v24.2-ubuntu',
          'v25.1-ubuntu',
          'v25.2-ubuntu'
```

---

I agree!